spec/spec_helper.rb

```diff
@@ -18,8 +18,13 @@
   config.before(:suite) do
     database_config = YAML.load(File.open("#{File.dirname(__FILE__)}/support/database.yml"))
     ActiveRecord::Base.establish_connection(database_config)
-    if Gem::Version.new(ActiveRecord::VERSION::STRING) >= Gem::Version.new("5.2.0")
-      ActiveRecord::MigrationContext.new("#{File.dirname(__FILE__)}/support/migrations").migrate
+
+    if Gem::Version.new(ActiveRecord::VERSION::STRING) == Gem::Version.new("5.2.0")
+      ActiveRecord::MigrationContext.new("#{File.dirname(__FILE__)}/support/migrations").migrate      
+    end
+    
+    if Gem::Version.new(ActiveRecord::VERSION::STRING) >= Gem::Version.new("6.0")
+      ActiveRecord::MigrationContext.new("#{File.dirname(__FILE__)}/support/migrations", ActiveRecord::SchemaMigration).migrate      
```

Comment:
As of https://github.com/rails/rails/commit/eb09f5997256061987f979b33b0c32d6b66a36a9 the default schema was set to `ActiveRecord::SchemaMigration`, so I think this value makes sense here.